test/NameBinding/scoped_import_overlay.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -enable-objc-interop -emit-module %S/Inputs/overlay.swift -module-name ClangModuleWithOverlay -I %S/Inputs/ClangModuleWithOverlay -o %t
+// RUN: %target-swift-frontend -typecheck %s -I %t
+
+// Make sure we can perform a scoped import on the overlay.
+import func ClangModuleWithOverlay.fromSwiftOverlay
+
+// ... as well as the underlying Clang module.
```

Comment:
Do we have any tests for doing this with import-as-member?

Reply:
@CodaFi Not that I can find, though we don't currently support scoped imports for anything other than top-level decls. I'll add a couple of negative test cases for that.

Thanks for the review!